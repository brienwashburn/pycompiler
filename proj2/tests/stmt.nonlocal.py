nonlocal a
nonlocal b, c, d, e, f, g
nonlocal b, c
