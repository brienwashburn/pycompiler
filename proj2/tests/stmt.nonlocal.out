(Module (Nonlocal a) (Nonlocal b c d e f g) (Nonlocal b c))
